hash.js

```diff
@@ -53,9 +53,14 @@ Hash.prototype.digest = function (enc) {
     this._block.fill(0)
   }
 
-  // to this append the block which is equal to the number l written in binary
-  // TODO: handle case where l is > Math.pow(2, 29)
-  this._block.writeInt32BE(l, this._blockSize - 4)
+  var maxUint32 = Math.pow(2, 32);
```

Comment:
Would it make sense to extract a constant (with module scope) like `var MAX_UINT32 = 0x100000000` instead of calling `Math.pow()` each time? Or would that be premature optimization?